### ISSUE TITLE
`cdf` is reversed for mirrored distributions

### ISSUE THREAD

**Issue by zmbc** [ihmeuw/risk_distributions#62]
MCVE:

```python
>>> import risk_distributions
>>> risk_distributions.risk_distributions.MirroredGumbel(risk_distributions.risk_distributions.MirroredGumbel.get_parameters(mean=5, sd=2.5)).cdf(10.0)
0.0006754912017839628
```

The return value should be very close to 1, not very close to 0.

**Comment by NathanielBlairStahn** [comment]
This behavior is documented (with pictures!) in this [notebook](https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/data_prep/risk_distribution/risk%20distribution%20comparison.ipynb), which is linked from this [note](https://vivarium-research.readthedocs.io/en/latest/model_design/general_reference_material/ensemble_distributions/index.html#where-is-the-code-for-gbd-s-ensemble-distributions:~:text=Given%20that%20there,a%20few%20exceptions%3A) in the ensemble distribution documentation.

**Comment by zmbc** [comment]
Nice! We'd still like to change this, though, right?

**Comment by NathanielBlairStahn** [comment]
> Nice! We'd still like to change this, though, right?

Yes, we should definitely fix it! I was just linking to the info we already had about this issue.

**Comment by aflaxman** [comment]
The docs Nathaniel pointed to also link into a slack conversation.  Can someone move that to a more permanent location if it is still relevant?

**Comment by patricktnast** [comment]
Closed via #70
